<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuoteFormWindow</class>
 <widget class="QDialog" name="QuoteFormWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Quote</string>
  </property>
  <widget class="QLineEdit" name="quoteeText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Quote</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Quotee</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="quoteText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>641</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>320</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>320</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
